### ISSUE TITLE
[UX] Hide Revision Log Message field when new revision is not being created

### ISSUE THREAD

**Issue by ghost** [backdrop/backdrop-issues#4164]
In a revision-enabled node in the Revision Information vertical tab, there's the Create New Revision checkbox and the Revision Log Message textarea. The textarea should be hidden (via FAPI States) until the checkbox is checked.

![IMG-20191027-071215.jpg](https://i.postimg.cc/k4xht7Sw/IMG-20191027-071215.jpg)

**Comment by klonos** [comment]
I agree 👍 ...especially makes sense when on mobile mode, as in the screenshot (less scrolling -> better UX).

**Comment by docwilmot** [comment]
Maybe not? Notice JS states is already involved: if you start typing in the textarea, that automatically checks the checkbox. Saves you a click.

**Comment by jenlampton** [comment]
I wonder: if revisions are enabled on a node, what percentage of time are they used? 

If less than 50%, I'd rather we hide the text-area and only show it when the box is checked. If more than 50% of the time, I'd prefer we show the text area and check the box by default.
